### PR TITLE
Merge Choropleth into Master

### PR DESCRIPTION
**Pull request by IanCostello**

### CODE REVIEW

**Comment by IanCostello** [comment]
Introduces a new (in development) choropleth map to DataCommons. The map exists solely for US geos, but works for any possible statistical variables. You can click into sub-geographies in the map to see a more detailed view of a region and click out of it with breadcrumbs. There is also further information of hover.

Screen Shot 2020-08-09 at 12 37 24 PM

File: server/main.py [hunk 1]
```diff
@@ -52,6 +52,20 @@
 }
 
 
+def get_place_args(get_values):
```
Comment (beets) [inline server/main.py:55]: please delete this function

File: static/data/hierarchy_statsvar.json [hunk 2]
```diff
@@ -129527,7 +129527,7 @@
       ],
       "l": "1,000,000 - 1,499,999 $",
       "t": "v",
-      "e": "USDollar1000000To1499999",
+      "e": "USDollar1000000Onwards",
```
Comment (beets) [inline static/data/hierarchy_statsvar.json:129530]: please undo this change

File: server/main.py (hunk 1 above)
Comment (IanCostello) [reply server/main.py:55]: Removed

File: static/data/hierarchy_statsvar.json (hunk 2 above)
Comment (IanCostello) [reply static/data/hierarchy_statsvar.json:129530]: Reverted

**Comment by IanCostello** [comment]
Removed erroneous files from the PR. @shifucun @beets
